README.md

```diff
@@ -17,12 +17,23 @@ Requirements
 ============
 
 Supported versions:
-* Python 2.6
+* Python 2.6 (currently not supported on AppVeyor)
```

Comment:
It does not cost us much effort to support it, the only problem is finding out how to configure AppVeyor with the right compiler (we should compile the python wheels with the same compiler used to compile that version of the python interpreter)